capstone/src/main/webapp/index.html

```diff
@@ -2,17 +2,84 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
     <title>STEP Capstone</title>
-    <link rel="stylesheet" href="style.css" />
-    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
     <link
       href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css"
       rel="stylesheet"
     />
     <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="style.css" />
```

Comment:
Shouldn't our stylesheet be on top of the material and other stylesheets? That way we don't overwrite what they may have since HTML is read from top to bottom, so anything below the material and google stylesheets will be overwritten by our own stylesheet. Or are you doing that for that same reason? Just checking your reasoning.

Reply:
Yeah, I am doing this so that I can change the styles form what material.io has. Some of the changes I needed to make would have needed a CSS preprocessor in order to implement properly.

Reply:
Alright perfect.